storage/src/tests/CMakeLists.txt

```diff
@@ -32,5 +32,6 @@ vespa_add_executable(storage_testrunner_app TEST
 
 vespa_add_test(
     NAME storage_testrunner_app
+    NO_VALGRIND
```

Comment:
Was this intentional? Do we really want to run without valgrind? @baldersheim

Reply:
it was intentional - that test suite took a really long time with valgrind, moving to gtest will fix this also i guess...

Reply:
I am not comfortable running storage tests without valgrind. How much of an impact does this really have compared to the rest of the build process?

Reply:
Then I suggest we run storage tests with valgrind as before. @arnej27959 can you fix?